tests/integration/integration_nodejs_test.go

```diff
@@ -823,7 +823,7 @@ func TestStackReferenceSecretsNodejs(t *testing.T) {
 			{
 				Dir:             filepath.Join(d, "nodejs", "step2"),
 				Additive:        true,
-				ExpectNoChanges: true,
+				ExpectNoChanges: false,
```

Comment:
Can just remove the field, it defaults to false.

Reply:
Done